NitroxServer/ConsoleCommands/Abstract/Type/TypeBoolean.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using NitroxServer.Exceptions;
+
+namespace NitroxServer.ConsoleCommands.Abstract.Type
+{
+    public class TypeBoolean : Parameter<bool?>, IParameter<object>
+    {
+        private static readonly IList<string> noValues = new List<string>
+        {
+            bool.FalseString,
+            "no",
+            "off"
+        };
+
+        private static readonly IList<string> yesValues = new List<string>
+        {
+            bool.TrueString,
+            "yes",
+            "on"
+        };
+
+        object IParameter<object>.DefaultValue => null;
+        object IParameter<object>.Read(string arg) => Read(arg);
```

Comment:
Indeed `Read` is already defined in the abstract class but to be added in our generic list of parameters, we need bool, float, int to inherit from `Parameter<object>`, Multiple inheritance isn't allowed in C# so we need to introduce a covariant interface

Reply:
It's needed because it needs to box value-types like `int` and `bool` to be a reference so that it can be stored in the `List<IParameter<object>>` variable. However, it's not needed for types that already have `object` as a base type.